<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>简易日历倒计时应用</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QLabel" name="positionLabel">
        <property name="text">
         <string>位置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="weatherLabel">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="yearViewButton">
        <property name="text">
         <string>年视图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="monthViewButton">
        <property name="text">
         <string>月视图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="weekViewButton">
        <property name="text">
         <string>周视图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addEventButton">
        <property name="text">
         <string>添加事件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteEventButton">
        <property name="text">
         <string>删除事件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showCountdownButton">
        <property name="text">
         <string>最小化</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="countdownLabel">
      <property name="text">
       <string>倒计时显示区域</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="eventListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menud">
    <property name="title">
     <string>任务</string>
    </property>
    <addaction name="actionffra"/>
    <addaction name="actioncdc"/>
    <addaction name="actiondr"/>
   </widget>
   <addaction name="menud"/>
  </widget>
  <action name="actionffra">
   <property name="text">
    <string>搜索 </string>
   </property>
  </action>
  <action name="actioncdc">
   <property name="text">
    <string>任务1</string>
   </property>
  </action>
  <action name="actiondr">
   <property name="text">
    <string>任务2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
